### PR TITLE
Stop using `bower`

### PR DESCRIPTION
**Pull request by adamzap**
`bower` is maintained but not recommended for use by its developers. Our `bower.json` file now only includes two dependencies: `jquery` and `jquery-flot`. At this point, `bower` is not doing much to serve the project.

This patch makes the following primary changes:

- Remove `bower.json`
- Remove `package.json`, which only included `bower`
- Put the minified source files for `jquery` and `jquery-flot` directly in `static/js/lib/`
- Remove the project directories for `jquery` and `jquery-flot`.

The following smaller changes are also included:

- Remove `Makefile` rules related to `bower` and `jquery-flot`
- Update the "JavaScript libraries" section of the `README.rst`
- Remove steps related to `package.json` from  `Dockerfile`

Part of #1827

### CODE REVIEW

**Review by pauloxnet — APPROVED** [review]
LGTM

**Review by bmispelon — COMMENTED** [review]
This looks great (as usual), I just found one thing that I think can be cleaned up further.

File: Dockerfile [hunk 1]
```diff
@@ -41,10 +41,6 @@ RUN apt-get update \
         zlib1g-dev \
     && rm -rf /var/lib/apt/lists/*
 
-# install node dependencies
-COPY ./package.json ./package.json
-RUN npm install
```
[the remark this answers is not in the document]
Comment (pauloxnet) [reply Dockerfile:46]: Good catch. :eyes: 

I checked the code and the same is true for the README.

You can remove the `npm install` in the `README.rst` (L3)
https://github.com/django/djangoproject.com/blob/main/README.rst?plain=1#L23

[the remark this answers is not in the document]
Comment (adamzap) [reply Dockerfile:46]: Sorry I missed that! I pushed 4628411 for the `npm` references and noticed 147cd76 as well.

**Review by bmispelon — APPROVED** [review]
Excellent 👍🏻

**Comment by adamzap** [comment]
Oh wow, ok. Do you want me to delete the line and push to this branch or a new branch?

**Comment by bmispelon** [comment]
> Oh wow, ok. Do you want me to delete the line and push to this branch or a new branch?

I'm fine with either. Can a PR be reopened after it's been merged?

(Also I opened a new issue to try and prevent this type of issue from happening in the future: #1831 )

**Comment by adamzap** [comment]
I don't think I can re-open this PR. I'll open a new one.

**Pull request merged**